src/core/mempool.rs

```diff
@@ -103,6 +108,21 @@ pub enum MemPoolDropReason {
     TOO_EXPENSIVE,
 }
 
+pub struct ConsiderTransaction {
+    /// Transaction to consider in block assembly
+    pub tx: MemPoolTxInfo,
+    /// If `update_estimator` is set, the iteration should update the estimator
+    /// after considering the tx.
+    pub update_estimate: bool,
+}
+
+enum ConsiderTransactionResult {
```

Comment:
Not a bug, but a style question -- is there a reason these are all caps?

Reply:
Defunct enum style memory on the part of the author. Will change to the rust-y standard.